src/test/resources/liquibase/harness/change/expectedSnapshot/databricks/createTable.json

```diff
@@ -1,17 +1,50 @@
 {
   "snapshot": {
+    "created": ".*",
+    "database": {
+      "productVersion": "3.1.1",
```

Comment:
plz remove productVersion, majorVersion, minorVersion fields from expected json files.  We don't want IT test to fail when Databricks changes the version